rasterio/rio/info.py

```diff
@@ -99,6 +122,48 @@ def info(ctx, input, aspect, indent, namespace, meta_member, verbose, bidx,
                         float(band.mean())))
                 elif meta_member == 'checksum':
                     click.echo(str(src.checksum(bidx)))
+
+                elif meta_member == 'gcps':
+                    gcps, crs = src.gcps
+
+                    if geojson_type == 'collection':
+                        features = [p.__geo_interface__ for p in gcps]
+                        if projection == 'geographic' or precision >= 0:
+                            prec = 7 if precision == -1 else precision
+                            dst_crs = 'epsg:4326' if projection == 'geographic' else crs
```

Comment:
Oh and have you thought about adding a CRS object to geojson?  :trollface: This case might actually warrant a foreign crs member or a crs feature property so that we don't lose information.